doc/polycli_ulxly_deposit-claim.md

```diff
@@ -0,0 +1,54 @@
+# `polycli ulxly deposit-claim`
+
+> Auto-generated documentation.
+
+## Table of Contents
+
+- [Description](#description)
+- [Usage](#usage)
+- [Flags](#flags)
+- [See Also](#see-also)
+
+## Description
+
+Make a uLxLy claim transaction
+
+```bash
+polycli ulxly deposit-claim [flags]
+```
+
+## Usage
+
+Make a uLxLy claim transaction
+## Flags
+
+```bash
+      --bridge-address string       The address of the bridge contract.
+      --bridge-service-url string   The RPC endpoint of the bridge service component.
+      --claim-address string        The address that is receiving the bridged asset.
+      --claim-index string          The deposit count, or index to initiate a claim transaction for. (default "0")
+      --claim-network string        The network ID of the origin network. (default "0")
+      --gas-limit uint              The gas limit for the transaction. (default 300000)
+  -h, --help                        help for deposit-claim
+      --private-key string          The private key of the sender account.
+      --rpc-url string              The RPC endpoint of the destination network (default "http://127.0.0.1:8545")
+```
```

Comment:
We'll be able to support `claimMessage` easily by adding a flag (for example `--asset` or `--message`) as it shares the same interface as `claimAsset`.